CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ CMAKE_DEPENDENT_OPTION(CARIBOU_OPTIMIZE_FOR_NATIVE "Build with -march=native to
 
 # SOFA option
 find_package(Threads QUIET)
-find_package(SOFA COMPONENTS SofaFramework QUIET)
+find_package(SOFA COMPONENTS Sofa.Config REQUIRED)
```

Comment:
Why using Required here ? Isn't `CARIBOU_WITH_SOFA` dealing with the case where SOFA cannot be found ?